lib/oban/engines/lite.ex

```diff
@@ -128,9 +128,11 @@ defmodule Oban.Engines.Lite do
 
     staged = Repo.all(conf, select_query)
 
-    Repo.update_all(conf, where(Job, [j], j.id in ^Enum.map(staged, & &1.id)),
-      set: [state: "available"]
-    )
+    unless staged == [] do
+      Repo.update_all(conf, where(Job, [j], j.id in ^Enum.map(staged, & &1.id)),
+        set: [state: "available"]
+      )
+    end
```

Comment:
Let's invert the conditional from "unless something is empty" to "if something is there":

```suggestion
    if Enum.any?(staged) do
      Repo.update_all(conf, where(Job, [j], j.id in ^Enum.map(staged, & &1.id)),
        set: [state: "available"]
      )
    end
```